infrastructure/zk/src/database.ts

```diff
@@ -53,10 +53,20 @@ export async function setup() {
         // Remote database, we can't show the contents.
         console.log(`WARNING! Using prod db!`);
     }
-    await utils.spawn('cargo sqlx database create');
-    await utils.spawn('cargo sqlx migrate run');
-    if (process.env.DATABASE_URL!.startsWith(localDbUrl)) {
-        await utils.spawn('cargo sqlx prepare --check -- --tests || cargo sqlx prepare -- --tests');
+    if (process.env.TEMPLATE_DATABASE_URL !== undefined) {
+        // Dump and restore from template database (simulate backup)
```

Comment:
Maybe it makes sense to add something like
```js
console.log(`Template DB URL provided. Creating a DB via dump from ${process.env.TEMPLATE_DATABASE_URL}`);
```

Reply:
done